chapters/template-chapter/template-topic/reading/README.md

```diff
@@ -0,0 +1,24 @@
+# Template Chapter
+
+This is an instance of a lab placeholder.
+This will be used to showcase the different highlights of the syntax.
+
+This is another sentence
+
+In order to use links to to outside resources, you can add them as [such](http://example.com).
+
+This is a list of all the best cuisines in the world
+- Indian
+- Italian
+- French
+
+In order to make emphasize keywords, you can use **bold** words.
+To suggest a more metaphorical and less literal meaning of a phrase, you can use _italic_ words.
+
+## Section the first
+
+For each skill learned in the lab, you will use a subsection which will touch up on new concepts.
+
+For integrating figures, you will use `![Example SVG](./media/app-os-cpu-interaction.svg)`.
+
+![Example SVG](../media/app-os-cpu-interaction.svg)
```

Comment:
It's not the image, but the example above it. It might be a bug in Docusaurus, I need to investigate.
Since the real location of the image is one directory up, I think it is safe to also use that as an example.

Reply:
I solved this by updating the path.